K2Bridge.Tests.UnitTests/JsonConverters/AggregationConverterTests.cs

```diff
@@ -73,143 +73,150 @@ public class AggregationConverterTests
-                            FieldName = "DestCountry",
-                            FieldAlias = "_2",
-                            Metric = "count()",
-                            SortFieldName = "_count",
-                            SortOrder = "desc",
+                    PrimaryAggregation = new TermsAggregation
+                    {
+                        Field = "DestCountry",
+                        Key = "2",
+                        Order = new TermsOrder {
+                            Field = "_count",
+                            Sort = "desc",
                         },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                    }
+                    },
+                    SubAggregations = new AggregationDictionary(),
                 },
             },
         };
 
-        private static readonly Aggregation ExpectedValidCardinalityAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedValidAvgAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
                 {
-                    { "2", new Aggregation() {
-                        PrimaryAggregation = new CardinalityAggregation {
-                            FieldAlias = "_2",
-                            FieldName = "metric",
-                            },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                        }
+                    PrimaryAggregation = new AverageAggregation
+                    {
+                        Field = "metric",
+                        Key = "2",
                     },
+                    SubAggregations = new AggregationDictionary(),
                 },
+            },
         };
 
-        private static readonly Aggregation ExpectedValidAvgAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedNoFieldsAvgAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
                 {
-                    { "2", new Aggregation() {
-                        PrimaryAggregation = new AvgAggregation {
-                            FieldAlias = "_2",
-                            FieldName = "metric",
-                            },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                        }
+                    PrimaryAggregation = new AverageAggregation
+                    {
+                        Field = null,
```

Comment:
metric or null?

Reply:
Tests order has been changed to default one.